ocamldoc/odoc_print.ml

```diff
@@ -95,9 +95,8 @@ let simpl_class_type t =
         let cs =
           { Types.csig_self = self;
             csig_self_row = self_row;
-            csig_vars = Types.Vars.empty ;
-            csig_meths = Types.Meths.empty ;
-            csig_inher = []; }
+            csig_vars = Types.Vars.empty;
+            csig_meths = Types.Meths.empty; }
         in
```

Comment:
`csig_inher` was intended to provide information about the inheritance hierarchy of a class type.
This is certainly useful information, and actually I was convinced that ocamldoc used it.
I believe Maxence asked me to add it.
Before removing it, it might be worth thinking again whether somebody has a use for it.